<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>641</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="3">
       <widget class="QPushButton" name="btn_continue">
        <property name="text">
         <string>Continue</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="cmb_set_param">
        <item>
         <property name="text">
          <string>Set 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Set 2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_set_of_param">
        <property name="text">
         <string>Set of parameters:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbl_parameters">
        <property name="text">
         <string>Parameters:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QTableWidget" name="tbl_parameters"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="cmb_element_type">
        <item>
         <property name="text">
          <string> -- </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blank Item</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TX - Transmitter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RX - Receiver</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Path Loss</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_elementtype">
        <property name="text">
         <string>Element Type:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="rdl_yes">
          <property name="text">
           <string>Yes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdl_no">
          <property name="text">
           <string>No</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_element_name">
        <property name="text">
         <string>Element Name:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="lbl_know_gainloss">
        <property name="text">
         <string>Do you know the Gain/Loss?</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPlainTextEdit" name="txt_element_name"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rdl_no</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>no_gain_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>554</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdl_yes</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>yes_gain_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>509</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_continue</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>continue_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>668</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>584</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>continue_clicked()</slot>
  <slot>yes_gain_clicked()</slot>
  <slot>no_gain_clicked()</slot>
 </slots>
</ui>
